src/test/resources/capabilities.yml

```diff
@@ -10,47 +10,13 @@ cloudDriver:
     enable: false
   common_capabilities:
     project: BrowserStack Demo Repository
-    buildPrefix: browserstack-examples-cucumber-testng
+    buildPrefix: browserstacktestng
```

Comment:
Keep it the same as the repo name. Easier to track.

Reply:
Corrected the Build Prefix name to browserstack-examples-cucumber-testng